pages/index.vue

```diff
@@ -0,0 +1,124 @@
+            of Deep Learning Researchers & Remote Sensing Scientists for the
+            development and exchange of best practices in Deep Learning for
+            Earth Objects Detection (Schools/Hospitals in this case!).
+          </p>
+        </div>
+        <div class="flex-1">
+          <img src="~/assets/school-childrens.png" alt="Geoqi" />
+        </div>
+      </div>
+
+      <!-- Bullets & Quick links-->
+      <div class="flex items-center py-10">
+        <!-- Bullets row 1 -->
+        <div class="flex-1">
+          <div class="flex-1 pb-10">
+            <div class="text-2xl mb-4">Codebase For Schools</div>
+            <div class="text-sm">
+              Here is the description and the link to the page contianing the
+              codebase.
+            </div>
+          </div>
+          <div>
+            <div class="text-2xl mb-4">Crowsdsource Validation Platform</div>
+            <div class="text-sm">
+              Here are the contents describing the Crowsdsource Validation
+              Platform a link to it.
+            </div>
+          </div>
+        </div>
+
+        <!-- Bullets row 2 -->
+        <div class="flex-1">
+          <div class="flex-1 pb-10">
+            <div class="text-2xl mb-4">Documentation For Schools</div>
+            <div class="text-sm">
+              Here are the contents describing the documentation and a link to
+              it.
+            </div>
+          </div>
+
+          <div>
+            <div class="text-2xl mb-4">Results Portal For Schools</div>
+            <div class="text-sm">
+              Here are the contents describing the Results Portal For Schools
+              and a link to it.
+            </div>
+          </div>
+        </div>
+
+        <!-- Quick Links -->
+        <div class="flex-1 text-center text-gray-700">
+          <div class="text-2xl mb-4">Quick Links</div>
+          <div class="mb-2 hover:text-blue-700 hover:underline">
+            <nuxt-link to="/about">What's this about?</nuxt-link>
+          </div>
+          <div class="mb-2 hover:text-blue-700 hover:underline">
+            <nuxt-link to="/objective">Objective</nuxt-link>
+          </div>
+          <div class="mb-2 hover:text-blue-700 hover:underline">
+            <nuxt-link to="/quick-start"
+              >Development Quickstart Guide</nuxt-link
+            >
+          </div>
+          <div class="mb-2 hover:text-blue-700 hover:underline">
+            <nuxt-link to="/results-portal">Rural Schools Portal</nuxt-link>
+          </div>
+          <div class="mb-2 hover:text-blue-700 hover:underline">
+            <nuxt-link to="/project-trials">Project trials</nuxt-link>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  import { defineComponent } from '@nuxtjs/composition-api';
+  export default defineComponent({
+    name: 'Index',
+  });
+</script>
```

Comment:
make this `landing.vue` instead of `index.vue` and let docus handle the routes for docs md file